devcontainer.md

```diff
@@ -0,0 +1,109 @@
+
+In VS Code, open a Terminal, you should be able to run the following commands:
+
+```
+python -m aries_cloudagent -v
+cd aries_cloudagent
+flake8 --max-line-length=90 --exclude=*/tests/** --extend-ignore=D202,W503 --per-file-ignores=*/__init__.py:D104
+black . --check
+```
+
+The first command should show you that `aries_cloudagent` module is loaded (ACA-Py). The others are examples of code quality checks that ACA-Py does on commits (if you have [`precommit`](https://pre-commit.com) installed) and Pull Requests.
+
+## Debugging
+
+To better illustrate debugging pytests and ACA-Py runtime code, let's add some run/debug configurations to VS Code. If you have your own `launch.json` and `settings.json`, please cut and paste what you want/need.
+
+```
+cp -R .vscode-sample .vscode
+```
+
+This will add a `launch.json`, `settings.json` and an ACA-Py configuration file: `multitenant.yml`. 
+
+### Pytest
+
+Pytest is installed and almost ready; however, we must build the test list. In the Command Palette, `Test: Refresh Tests` will scan and find the tests.
+
+See [Python Testing](https://code.visualstudio.com/docs/python/testing) for more details, and [Test Commands](https://code.visualstudio.com/docs/python/testing#_test-commands) for usage.
+
+*IMPORTANT*: our pytests include coverage, which will prevent the [debugger from working](https://code.visualstudio.com/docs/python/testing#_debug-tests). One way around this would be to have a `.vscode/settings.json` that says not to use coverage (see above). This will allow you to set breakpoints in the pytest and code under test and use commands such as `Test: Debug Tests in Current File` to start debugging.
+
+
+### ACA-Py
+
+Above, we added some run/debug configurations, one of which is to run our ACA-Py source code so we can debug it.
+
+In `launch.json` you will see:
+
+```
+        {
+            "name": "start acapy",
+            "type": "python",
+            "request": "launch",
+            "module": "aries_cloudagent",
+            "justMyCode": true,
+            "args": [
+                "start",
+                "--arg-file=${workspaceRoot}/.vscode/multitenant.yml"
+            ]
+        },
+```
+
+This will start your source code as a running ACA-Py instance, all configuration is in the `multitenant.yml` file. This is just a sample of a configuration, and we chose multi-tenancy so we can easily create multiple wallets/agents and have them interact.  Note that we are not using a database and are joining the ` http://test.bcovrin.vonx.io` ledger. Feel free to change to a local VON Network (by default, it would be `http://host.docker.internal:9000`) or another ledger. This is purposefully a very simple configuration.
```

Comment:
Could note ways to run this launch config (discussed in RC)

Reply:
addressed